### PR TITLE
CUMULUS-3098: Add task configuration to lzards-backup task to fail task

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-3098: LZARDS backup tasks does not result in error if a LZARDS timeout occurs](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3098)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [x] Integration tests

### CODE REVIEW

File: tasks/lzards-backup/schemas/config.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
     "cloudfrontEndpoint": {
       "description": "cloudfront endpoint URL/host. Required if urlType is cloudfront",
       "type": "string"
+    },
+    "failTaskWhenFileBackupFail": {
+      "description": "Indicates if the task will fail when file backup request fails",
+      "type": "boolean",
+      "default": false
```
Comment (botanical) [inline tasks/lzards-backup/schemas/config.json:19]: Can we add this new config value to the readme of the task too?

**Comment by jennyhliu** [comment]
> Should we add integration test too? Or is that not necessary 🤔

Good point.  Done

File: example/spec/parallel/lzardsBackup/LzardsBackupWorkflowSpec.js [hunk 2]
```diff
@@ -137,6 +145,17 @@ describe('The Lzards Backup workflow ', () => {
...
+      expect(backupStatus.length).toEqual(2);
+      for (let i = 0; i < backupStatus.length; i += 1) {
+        if (backupStatus[i].filename.endsWith('testGranule.jpg')) {
+          expect(backupStatus[i].status).toEqual('COMPLETED');
+        } else if (backupStatus[i].filename.endsWith('testGranuleFail.jpg')) {
+          expect(backupStatus[i].status).toEqual('FAILED');
+        } else {
+          fail(`unexpected backup file ${backupStatus[i].filename}`);
+        }
+      }
```
Comment (botanical) [inline example/spec/parallel/lzardsBackup/LzardsBackupWorkflowSpec.js:158]: ✨ Nice

File: example/spec/parallel/lzardsBackup/LzardsBackupWorkflowSpec.js [hunk 3]
```diff
@@ -333,4 +352,70 @@ describe('The Lzards Backup workflow ', () => {
...
+
+    afterAll(async () => {
+      await deleteExecution({ prefix, executionArn: workflowExecution.executionArn });
+    });
+
+    it('throws an error and fails', () => {
+      if (beforeAllFailed) fail(beforeAllFailed);
+      const errorCause = JSON.parse(lambdaOutput.cause);
+      expect(workflowExecution.status).toEqual('failed');
+      expect(errorCause.errorMessage).toContain('testGranuleFail.jpg did not have a checksum or checksumType defined');
```
Comment (botanical) [inline example/spec/parallel/lzardsBackup/LzardsBackupWorkflowSpec.js:418]: 👍

**Review by botanical — APPROVED** [review]
Thanks for addressing the feedback 🙇 looks good

**Pull request merged**